apps/aot/src/app/events/[year]/[day]/_components/comments/comment.tsx

```diff
@@ -105,14 +105,13 @@ export function Comment({
     updateReplyComment,
     deleteReplyComment,
     showLoadMoreRepliesBtn,
-    // @ts-ignore
   } = useCommentsReplies({
     enabled: showReplies,
     root,
     type,
     parentComment: comment,
     preselectedReplyId: hasPreselectedReply ? Number(replyId) : undefined,
-  });
+  } as UseCommentRepliesProps);
```

Comment:
[`@typescript-eslint/ban-ts-comment`](https://typescript-eslint.io/rules/ban-ts-comment): both the `// @ts-ignore` and the `as UseCommentRepliesProps` work around the type system. The type assertion (`as`) is much more direct and disables fewer type checks.

WIP blog post explaining them: https://github.com/LearningTypeScript/site/pull/143